### PR TITLE
audio_recv: doc updates.

### PR DESCRIPTION
**Pull request by dragonwolfsp**
# This pull request makes the folloing changes:

- **update code example for audio block usage.**
- **Add on_muxed_audio to event reference.**
- **Update whates new docs.**

### CODE REVIEW

**Comment by JessicaTegner** [comment]
Hi @blindelectron  CI is failing. I fixed the error on the upstream branch, so if you could  rebase onto the up-to-date audio_recv branch, we can get this squared away

File: docs/_static/csv/server_channel_n_file.csv [hunk 1]
```diff
@@ -5,3 +5,4 @@ on_server_update	teamtalk.Server	Called when a teamtalk.Server has updated its s
 on_server_statistics	teamtalk.Statistics	Called when a TeamTalk server sends it's statistics.
 on_file_new	teamtalk.RemoteFile	Called when a file has been uploaded to a channel.
 on_file_delete	teamtalk.RemoteFile	Called when a file has been deleted from a channel.
+on_muxed_audio	teamtalk.MuxedAudioBlock	called when a muxed audio block is recieved, this is used to get audio from all users in the channel at once.
```
Comment (JessicaTegner) [inline docs/_static/csv/server_channel_n_file.csv:8]: We are missing on_user_audio

File: teamtalk/audio.py [hunk 2]
```diff
@@ -3,14 +3,19 @@
...
 
         Example
-        ---------
+        -------
 
         .. code-block:: python3
 
+            import pyaudio
+            streams: dict = {}
+            p = pyaudio.PyAudio
```
Comment (JessicaTegner) [inline teamtalk/audio.py]: This example doesn't actually work...

**Review by JessicaTegner — CHANGES_REQUESTED** [review]
@blindelectron

**Comment by JessicaTegner** [comment]
@blindelectron  merge conflicts

**Comment by dragonwolfsp** [comment]
Conflicts fixed and review issues resolved.

**Pull request merged**